x/checkpoint/keeper/keeper.go

```diff
@@ -30,7 +30,7 @@ var (
 
 // ModuleCommunicator manages different module interaction
 type ModuleCommunicator interface {
-	GetAllDividendAccounts(ctx sdk.Context) []hmTypes.DividendAccount
+	GetAllDividendAccounts(ctx sdk.Context) []*hmTypes.DividendAccount
```

Comment:
@murthyvitwit @temaniarpit27 , this will work after the latest changes in the proto files right?